flink-cdc-common/src/main/java/com/ververica/cdc/common/event/AddColumnEvent.java

```diff
@@ -66,20 +66,20 @@ public static class ColumnWithPosition implements Serializable {
         private final ColumnPosition position;
 
         /** The added column lies in the position relative to this column. */
-        private final @Nullable Column existingColumn;
+        private final @Nullable String existedColumnName;
 
         /** In the default scenario, we add fields at the end of the column. */
         public ColumnWithPosition(Column addColumn) {
```

Comment:
This is a public API, could we avoid changing this method ?

Reply:
It's a `PublicEvolving` API, we change this because column type is necessary for sink to apply SchemaChange, and source now just pass an arbitrary column type to build it.